addons/sale_timesheet/report/timesheets_analysis_report.py

```diff
@@ -36,11 +36,18 @@ def _select(self):
             A.so_line AS so_line,
             A.timesheet_invoice_type AS timesheet_invoice_type,
             A.timesheet_invoice_id AS timesheet_invoice_id,
-            CASE WHEN A.order_id IS NULL THEN 0 ELSE A.unit_amount * SOL.price_unit END AS timesheet_revenues,
+            CASE WHEN A.order_id IS NULL THEN 0 ELSE A.unit_amount * SOL.price_unit * sol_product_uom.factor / a_product_uom.factor END AS timesheet_revenues,
             CASE WHEN A.order_id IS NULL THEN 0 ELSE A.unit_amount END AS billable_time
         """
 
     @api.model
     def _from(self):
+        """
+        A is a model coming from the parent class's _from function (atm account.analyic.line)
+        SOL.product_uom matching u.id enables factor retrieval of 1 for day, hours of day for hour unit of measure (uom)
+        likewise for uom_uom matching on a.product_uom_id enables factor retrieval of hours of day uom for all
```

Comment:
current version of the PR looks good to me, but this comment is not necessary I think. It's already knowledge we have by looking at the code.